manifests/config.pp

```diff
@@ -9,24 +9,24 @@
 
   file {
     default:
-      owner => 'root',
-      group => 'root',
+      owner => $puppetwebhook::webhook_user,
```

Comment:
I'd make the owner `root` (since it's in `/etc`) and mode `0640` so the group can still read it but if you run it as non-root you can't modify the actual file.

Reply:
I'll probably leave the user configurable since there will be an install from rpm and deb available sometime next month as well and those will have their own preset user/group settings. This way the defaults are still set to root for owner, but can be easily configurable for rpm/deb installs